src/app/donate/new-donate/page.tsx

```diff
@@ -1,22 +1,27 @@
 import CampaignCard from "@/components/donate/CampaignCard";
+import { Container, Grid } from "@radix-ui/themes";
 import { FUND_RAISERS } from "@/data/donate";
 
 const Page = () => {
   return (
-    <div>
-      {FUND_RAISERS.map((donate, i) => (
-        <CampaignCard
-          key={i}
-          imgSrc={donate.img}
-          imgAlt=""
-          title={donate.title}
-          raised={donate.percentage}
-          goal={donate.goal_price}
-          donateLink={donate.donate_link}
-          moreLink={donate.more_link}
-        />
-      ))}
-    </div>
+    <Container px="4">
+      <Grid gap="8" columns={{ xs: "2", md: "3" }} pt="4.5rem" pb="9rem">
```

Comment:
**suggestion (blocking):** It looks like the flex layout is forced to wrap at different points, and the cards get very thin at the 600px mark.  This can be solved by setting it so there's one column at the `xs` breakpoint and 2 at the `sm` breakpoint (along with the 3 at the `md` breakpoint).